samples/react-native/.eslintrc.json

```diff
@@ -37,7 +42,33 @@
         "args": "none"
       }
     ],
+    "func-names": "off",
     "radix": ["error", "as-needed"],
-    "prefer-destructuring": "off"
+    "prefer-destructuring": "off",
```

Comment:
Some relevant comments made on the base .eslintrc file:

1. Should be able to remove the following (defaults):
```
"no-restricted-globals": "off",
"no-restricted-properties": "off",
"max-len": "off",
"no-debugger": "error",
"no-empty": "error",
"no-fallthrough": "error",
```

2. "no-console" rule doing much with all of the allows?

Reply:
Removed, and removed `no-console`